rialto_airflow/dags/harvest.py

```diff
@@ -21,10 +21,11 @@
 sul_pub_key = Variable.get("sul_pub_key")
 
 # to artificially limit the API activity in development
+dev_limit = None
 try:
     dev_limit = int(Variable.get("dev_limit", default_var=None))
 except TypeError:
-    dev_limit = None
+    pass
```

Comment:
one of the few hoops i jumped through to make things less confusing for the type checker (otherwise it thinks `dev_limit` should be a non-optional `int`, since that was its first possible assignment).